pages/events/index.vue

```diff
@@ -161,6 +187,38 @@ export default class extends QiskitPage {
     return (this as any).filteredEvents.length === 0
   }
 
+  get optionsList (): any {
+    return (listType: string) => {
+      return this[listType].map((item) => {
+        return {
+          label: item,
+          value: item,
+          name: item
+        }
+      })
+    }
+  }
+
+  updateRegionFilters (selectedRegions) {
+    const { commit } = this.$store
+    const payload = {
+      filter: 'regionFilters',
+      filterValues: selectedRegions
+    }
+
+    commit('updateFilterSet', payload)
+  }
+
+  updateTypeFilters (selectedRegions) {
+    const { commit } = this.$store
+    const payload = {
+      filter: 'typeFilters',
+      filterValues: selectedRegions
+    }
+
+    commit('updateFilterSet', payload)
+  }
```

Comment:
I think that both can be merged into a single function with a new parameter `filter` as the `updateFilter` function does.
Also, if you replace the `selectedXXXX` variable name with `filterValues`, you can follow the `updateFilter` pattern in order to simplify reading.
I don't know if there is any way to merge them with `updateFilter`. I think that there is not a simple solution.

Reply:
Thanks for the guidance here, @korgan00